packages/mobile/src/screens/notifications-screen/NotificationsScreen.tsx

```diff
@@ -25,9 +25,9 @@ const styles = StyleSheet.create({
 /**
  * A component that renders a user's notifications
  */
-export const NotificationsScreen = () => {
+export const NotificationsScreen = memo(() => {
```

Comment:
Just curious what is triggering this without memo? The handlers object changing in context?

Reply:
No, I think the Drawer.Navigator `drawerContent` is just getting rerendered on navigation state changes, not sure why. But none of the props or the context value are changing so I added this memo to stop the rerender as high as possible